docs/Getting Started/Registering an Application/index.md

```diff
@@ -19,7 +19,7 @@ If the implementation of Core supports hash resumption, any time an item that ca
 
 ### OnPermissionChange
 
-After the app has registered, Core will check the policy table to see if the app has any permission updates.  You can find more information about [OnPermissionChange parameters](https://github.com/smartdevicelink/rpc_spec#onpermissionchange) in the RPC spec.
+After the app has registered, Core will check the policy table to see if the app has any permission updates.  You can find more information about [OnPermissionChange parameters](https://github.com/smartdevicelink/rpc_spec#onpermissionschange) in the RPC spec.
```

Comment:
```suggestion
After the app has registered, Core will check the policy table to see if the app has any permission updates.  You can find more information about [OnPermissionsChange parameters](https://github.com/smartdevicelink/rpc_spec#onpermissionschange) in the RPC spec.
```

Reply:
Changed all instances of `OnPermissionChange` in https://github.com/smartdevicelink/sdl_overview_guides/pull/72/commits/8ccd06238eec3909a54ce58ddc921b280110186d